nostr/event.py

```diff
@@ -1,10 +1,12 @@
+from binascii import unhexlify
 import time
 import json
 from dataclasses import dataclass, field
 from enum import IntEnum
 from typing import List
 from secp256k1 import PrivateKey, PublicKey
 from hashlib import sha256
+from nostr import bech32
```

Comment:
prefer relative imports
```suggestion
from . import bech32
```